client/js/components/document/ElementsAdminList.vue

```diff
@@ -282,18 +284,19 @@ export default {
       this.selectedFiles = selected
         .filter(node => node.nodeType === 'leaf')
         .map(el => el.nodeId)
+
       this.selectedElements = selected
         .filter(node => node.nodeType === 'branch')
         .map(el => el.nodeId)
     },
 
     /**
      * Callback that is executed whenever an item is dragged over a new target.
-     * Here it is used to cancel the drag action when dragging over singleDocument
+     * Here it is used to cancel the drag action when dragging over singleDocument (=!isBranch)
      * elements, hereby keeping folders above files.
      */
-    onMove ({ relatedContext }) {
-      return relatedContext.element.type !== 'singleDocument'
+    onMove (e) {
```

Comment:
@hwiem do You rember the purpose of that changes?

Reply:
my concern was to use a proper name instead of `e`.

Reply:
yeah. But I don't know what kind of Data is provided. If its an `event` or an `element`

Reply:
i see, so "e" is the thing they both have in common :)

Reply:
The event is documented here https://github.com/SortableJS/Sortable#options (-> `onMove`).
When committing this code I remember thinking that we should probably find a better solution before merging, because accessing `e.related.__vnode` seems very ugly :grimacing: